scripts/avocado

```diff
@@ -38,7 +38,7 @@ def handle_exception(*exc_info):
     try:
         import time
         prefix += time.strftime("%F_%T") + "-"
-        from avocado.core import data_dir
+        from avocado.core import data_dir    # pylint: disable=E0611
```

Comment:
The problem is that this file has the same name of the imported module. Pylint will add the current working directory on top of the python path, making the `scripts/avocado` to have precedence over system module `avocado`. Not sure there' a best solution here, so ack.